bin/oref0-online.sh

```diff
@@ -141,7 +141,7 @@ function check_ip {
         return 1
     else
         echo $PUBLIC_IP
-        touch /tmp/hasPublicIp
+        echo $PUBLIC_IP > /tmp/hasPublicIp
```

Comment:
`hasPublicIp` was a great file name when we were treating the file as a flag, but now that we're storing the value of the IP in the file, should we rename the file to something like `publicIp`?

Reply:
I don't care about the exact name. I just need the feature :) So feel free to change it ;)

Reply:
Like that?